keyboards/keychron/k10_pro/iso/rgb/info.json

```diff
@@ -1,133 +1,32 @@
 {
-    "keyboard_name": "Keychron K10 Pro",
-    "manufacturer": "Keychron",
-    "url": "https://github.com/Keychron",
-    "maintainer": "lalalademaxiya1",
     "usb": {
         "vid": "0x3434",
```

Comment:
```suggestion
```

Reply:
Resolved in https://github.com/Keychron/qmk_firmware/commit/71515b61904d8d31fdfdf4b0075759d6cd420b65.